hack/download-frozen-image-v2.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
we'll likely remove this as well once we have `docker-archive` transport and directly use skopeo probably

Reply:
We've now had `docker-archive` for a while, so we can drop this. :wink: